airmozilla/main/models.py

```diff
@@ -9,7 +9,7 @@
 from django.db import models
 from django.db.models import Q
 from django.dispatch import receiver
-from django.utils import timezone
+from django.utils.timezone import utc
```

Comment:
This diff should be removed. It should just be `from django.utils import timezone`